src/test/java/com/sigma/cinematicketpro/TestUtils.java

```diff
@@ -147,4 +154,27 @@ public static RegistrationRequest getValidRegistrationRequest() {
                 .password("secure_pass")
                 .build();
     }
+
+    public static List<TMDBGenre> getTMDBGenresList() throws IOException {
+        return mapJsonToObject("tmdb/genres/genres.json", new TypeReference<>() {});
+    }
+
+    public static Map<Integer, TMDBGenre> getTMDBGenresIdGenreMap() throws IOException {
+        return getTMDBGenresList().stream()
+                .collect(toMap(TMDBGenre::getId, identity()));
+    }
+
+    public static List<TMDBMovie> getTMDBMoviesWithMappedGenresTitles() throws IOException {
+        return mapJsonToObject("tmdb/movies/trending-movies-mapped-genres-titles.json", new TypeReference<>() {});
+    }
+
+    public static List<TMDBMovie> getTMDBMoviesWithoutGenresTitles() throws IOException {
```

Comment:
you need it in one test - I d rather make it private method in it

Reply:
fixed